cmd/tke-installer/app/installer/manifests/thanos/thanos.yaml

```diff
@@ -86,7 +90,7 @@ metadata:
   name: thanos-receive
   namespace: tke
 spec:
-  replicas: 1
+  replicas: 3
```

Comment:
This com has 3 replicas, while other coms have 2. Cloud you talk about the reason?